tools/drdeploy.c

```diff
@@ -970,6 +970,10 @@ read_tool_file(const char *toolname, const char *dr_root, dr_platform_t dr_platf
             _snprintf(alt_client, alt_size, "%s/%s", dr_root,
```

Comment:
In the PR description, please include `Issue: #5153`, as this PR is relevant to that issue.